.bazelrc

```diff
@@ -1,4 +1,5 @@
 common --noenable_bzlmod
+common --lockfile_mode=off
```

Comment:
Since `MODULE.bazel.lock` is in `.gitignore`, we can keep the default mode (`update`) for better local incremental build times.